md/release-notes.md

```diff
@@ -56,6 +56,21 @@ This helps tracking the processing when parallel requests are in progress.
 ```
 
 
+## No more migration between maintenance versions of Bonita
+Starting with Bonita 7.11.0, it is not necessary to run the Bonita migration tool to migrate between maintenance versions of the Bonita Platform (between 7.11.**0** and 7.11.**1**).
```

Comment:
```suggestion
Starting with Bonita 7.11.0, it is not necessary to run the Bonita migration tool to migrate between maintenance versions of Bonita Runtime (between 7.11.**0** and 7.11.**1**, for instance).
```

Reply:
The message to share is that there is no **migration**, only **upgrade** of binaries